### DISCUSSION TITLE
Boot from degraded ZFS mirror

### DISCUSSION THREAD

**Discussion by beremour** [zbm-dev/zfsbootmenu#548] [Questions]
Why zfsbootmenu can't boot from degraded mirror ? I am getting infinite loop of "wating for menu..". Zpool can be imported and files can be red.

**Comment by ahesford** [comment]
If you can import the pool with just `zpool import -N -o readonly <pool>`, ZFSBootMenu should have no problems booting from a degraded mirror. If it requires additional options like `-f`, ZFSBootMenu will not inject those without being told to. If you are sure you want to force the import, `zbm.import_policy=force` should get you there.

**Comment by beremour** [comment]
Thank you very much! I've only tried -f. Will test -N import also. Where ```zbm.import_policy=force``` should be placed ?

**Comment by zdykstra** [comment]
You shouldn't need to force import a degraded mirror. What is the full message you're seeing on the screen, and what's the output of `zreport` in a recovery shell?

**Reply by zdykstra** [reply]
I've just confirmed that `zpool` will import a degraded mirror with out any issues, so by extension ZFSBootMenu will as well. There's something else wrong here.
